e2e/async-regenerator/__tests__/test.js

```diff
@@ -6,6 +6,6 @@
  */
 
 test('dummy test', async () => {
-  const value = await Promise.resolve(1);
+  const value = 1;
```

Comment:
if there is no await anymore then you don't need this async either

but
this change defeats the purpose of an async test example
the goal was to demonstrate how async tests are being handled in jest

Reply:
specifically, it verifies transforming it to use generators work, so not having `await` defeats the whole purpose of the test, like @Smrtnyk says.

https://github.com/facebook/jest/blob/2bc72c68b0e101c4212626d48c20a3b08e5940e6/e2e/async-regenerator/babel.config.js#L10